experimental/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to experimental packages in this project will be documented
 
 ## Unreleased
 
+* feat(instrumentation): add new `_setMeterInstruments` protected method that update the meter instruments every
```

Comment:
Nit: this should be categorized as an enhancement. Also, empty lines need to be removed.

Reply:
fixed, ty :)